packages/configuration-backend/src/service.ts

```diff
@@ -318,3 +319,19 @@ export const getSubscriptionTermsOfServiceBinary = async (p: {
     })
   }
 }
+
+export const parseMerchantIdFromFirstOrderItem = (order: {
```

Comment:
This implementation is copied from your PR KYV-699. I will merge this after your PR has been merged, so I can sync the branches an then there will not be any conflicts.